inspectors/view/qml/notation/NotationInspectorView.qml

```diff
@@ -120,5 +121,23 @@ FocusableItem {
             popupAvailableWidth: root.width
             model: root.model ? root.model.modelByType(Inspector.TYPE_CLEF) : null
         }
+
+        TextFrameSettings {
+            popupPositionX: mapToGlobal(grid.x, grid.y).x - mapToGlobal(x, y).x
+            popupAvailableWidth: root.width
+            model: root.model ? root.model.modelByType(Inspector.TYPE_TEXT_FRAME) : null
+        }
+
+        VerticalFrameSettings {
+            popupPositionX: mapToGlobal(grid.x, grid.y).x - mapToGlobal(x, y).x
+            popupAvailableWidth: root.width
+            model: root.model ? root.model.modelByType(Inspector.TYPE_VERTICAL_FRAME) : null
+        }
+
+        HorizontalFrameSettings {
+            popupPositionX: mapToGlobal(grid.x, grid.y).x - mapToGlobal(x, y).x
+            popupAvailableWidth: root.width
+            model: root.model ? root.model.modelByType(Inspector.TYPE_HORIZONTAL_FRAME) : null
+        }
```

Comment:
here should be additional line with `onPopupContentHeightChanged: updateContentHeight(popupContentHeight)`